### PR TITLE
Add Tuya `TS0002` 2 gang switch variant

### PR DESCRIPTION
**Pull request by javicalle**
New `Switch_2G_Var03` variation for the `TS0002` Tuya 2 gang devices.

Fix: #2272

### CODE REVIEW

File: zhaquirks/tuya/ts000x.py [hunk 1]
```diff
@@ -322,6 +322,98 @@ class Switch_2G_Metering(EnchantedDevice, CustomDevice):
     }
 
 
+class Switch_2G_Var03(EnchantedDevice, CustomDevice):
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts000x.py:325]: I think just `EnchantedDevice` is enough, as that also extends `CustomDevice`. But I see that it's also done like this in other places, so I guess it's fine too

File: zhaquirks/tuya/ts000x.py [hunk 2]
```diff
@@ -322,6 +322,98 @@ class Switch_2G_Metering(EnchantedDevice, CustomDevice):
...
+    """Tuya 2 gang (variation 03))."""
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts000x.py:326]: Second `)` can probably be removed

Comment (javicalle) [reply zhaquirks/tuya/ts000x.py]: Done: https://github.com/zigpy/zha-device-handlers/pull/2296/commits/0dc12e01f3c47aa4ddc24b6c469e77c0a7b3a9d8

File: zhaquirks/tuya/ts000x.py (hunk 1 above)
Comment (javicalle) [reply zhaquirks/tuya/ts000x.py:325]: Yes, you're right. And have warned me already 😞 
I'll update all of them (in this file) in another PR

Comment (MattWestb) [reply zhaquirks/tuya/ts000x.py:325]: I think some one was doing most or all of them 😞

**Review by TheJulianJES — APPROVED** [review]

**Comment by javicalle** [comment]
Thanks Julian

**Pull request merged**